### PR TITLE
Added useful Github resources

### PR DESCRIPTION
**Pull request by hkasera**
I have added few very useful Github resources that helped me a lot. I have raised an issue (#28) for the same. I hope this helps.

### CODE REVIEW

File: learn_to_program.md [hunk 1]
```diff
@@ -152,6 +152,10 @@
 *   [Git-it - intro Git/Github tutorial](https://github.com/jlord/git-it) (includes Traditional Chinese version, requires local install)
 *   [Learn Git Branching](http://pcottle.github.io/learnGitBranching/) (interactive tutorial)
 *   [GitHub - Git Cheat Sheet](https://training.github.com/kit/downloads/github-git-cheat-sheet.pdf) (pdf)
+*   [Github - Pull requests](https://training.github.com/classnotes/2014-05-28-free-class-pull-requests)
+*   [Understanding Git Concepts](Understanding Git Conceptually)
```
Comment (jig08) [inline learn_to_program.md]: Hey @hkasera 
Could you please add a valid URL here ?
This gets directed to [https://github.com/hkasera/guidelines-resources/blob/git-useful-links/Understanding%20Git%20Conceptually](https://github.com/hkasera/guidelines-resources/blob/git-useful-links/Understanding%20Git%20Conceptually) which shows a **404 Page Not Found** Error

**Comment by hkasera** [comment]
Hi @jig08, 

Apologies for missing that. I have updated the link and now it points to the correct URL (http://www.sbf5.com/~cduan/technical/git/)

**Comment by hkasera** [comment]
Thanks @zassmin and @jig08  :)

**Comment by jig08** [comment]
Anytime @hkasera :)

**Pull request merged**